bundle/Resources/public/js/views/plugins/cof-selectcontenttypeplugin.js

```diff
@@ -16,6 +16,70 @@ YUI.add('cof-selectcontenttypeplugin', function (Y) {
     Y.cof.Plugin.SelectContentType = Y.Base.create('selectContentType', Y.eZ.Plugin.ContentCreate, [], {
         initializer: function () {
             this.onHostEvent('*:fetchContentTypes', this._getContentTypes, this);
+            this.onHostEvent('*:fetchSuggestedLocations', this._getSuggestedLocations, this);
+        },
+
+        /**
+         * Get the suggested locations
+         *
+         * @protected
+         * @method _getSuggestedLocations
+         * @param event {Object} event facade
+         */
+        _getSuggestedLocations: function (event) {
+            var contentTypeId = '/' + event.newVal.get('identifier');
+
+            Y.io(this.get('host').get('app').get('apiRoot') + 'api/ezp/v2/contentonthefly/locations' + contentTypeId, {
+                on: {
+                    success: Y.bind(function (id, xhr) {
+                        var locationSchema = {resultListLocator: 'Location', resultFields: [{key:'href', locator: '@href'}]},
+                            locations = Y.DataSchema.XML.apply(locationSchema, xhr.responseXML);
```

Comment:
Add `Accept: application/vnd.ez.api.LocationList+json` header to the request. You'll get JSON response:

Example JSON: 
```json
{
    "LocationList": {
        "_media-type": "application\/vnd.ez.api.LocationList+json",
        "_href": "\/api\/ezp\/v2\/contentonthefly\/locations\/blog_post",
        "Location": [
            {
                "_media-type": "application\/vnd.ez.api.Location+json",
                "_href": "\/api\/ezp\/v2\/content\/locations\/1\/2"
            },
            {
                "_media-type": "application\/vnd.ez.api.Location+json",
                "_href": "\/api\/ezp\/v2\/content\/locations\/1\/43"
            }
        ]
    }
}
```